background/services/chain/index.ts

```diff
@@ -148,7 +148,7 @@ export default class ChainService extends BaseService<Events> {
       blockPrices: {
         schedule: {
           periodInMinutes:
-            Number(process.env.BLOCKNATIVE_POLLING_FREQUENCY) / 60,
+            Number(process.env.BLOCKNATIVE_POLLING_FREQUENCY) / 60 ?? 2,
```

Comment:
I don't think this will have the desired effect, as the `??` binds to the result of `Number(process.env.BLOCKNATIVE_POLLING_FREQUENCY) / 60`, which will never be `undefined`. Instead, I think we want this:

```suggestion
            Number(process.env.BLOCKNATIVE_POLLING_FREQUENCY ?? "2") / 60,
```

Reply:
True that, I have corrected that in network fees PR. Thanks!